docs/technical_reference/unit_models/zero_order_unit_models/automate_rst_file_wt3.py

```diff
@@ -173,7 +164,7 @@ def grab_unit_components_feed(unit_class):
     m.fs.props = WaterParameterBlock(
         solute_list=[
             "toc",
-            "tss",
+            "tkn" "tss",
             "cod",
             "tds",
             "nitrogen",
```

Comment:
Without knowing the context, is it possible a comma is missing here?

```suggestion
            "tkn",
            "tss",
            "cod",
            "tds",
            "nitrogen",
```

Reply:
Yes, thx for catching that!